l2gv2/embedding/embeddings.py

```diff
@@ -51,32 +51,31 @@ def fit(
 
     def embed(
         self,
-        patch_data,
-        dim=100,
-        hidden_dim=32,
+        patch_data: list,
+        dim: int=100,
+        hidden_dim: int=32,
         decoder=None,
-        device="cpu"
+        device: str="cpu"
```

Comment:
Remove this? if we are keeping, can make a stricter type: Literal["cpu", "gpu"] to indicate valid choices